CMSgov/mdct-mcr
# Promote MCR main to val

## Description
Promote main to val for design to verify. 

Key changes:
- Add number field with various masks (visible at `/mcpar/test`)
- Add BSS Entities page `/mcpar/program-information/add-bss-entities`
- Update homepage text and treatment
- Update app width on desktop

### How to test
Check [mdctmcrdev.cms.gov](mdctmcrdev.cms.gov) and see if you like it (can compare to [mdctmcrval.cms.gov](mdctmcrdev.cms.gov))

## Code author checklist

## Reviewer checklist (two different people)
